src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -501,9 +503,10 @@ void LiveFunctionsDataView::OnDataChanged() {
     const FunctionInfo* function_info_from_capture_data;
 
     function_info_from_capture_data =
-        orbit_client_data::FindFunctionByModulePathBuildIdAndVirtualAddress(
-            *module_manager, instrumented_function.file_path(),
-            instrumented_function.file_build_id(),
+        orbit_client_data::FindFunctionByModuleIdentifierAndVirtualAddress(
+            *module_manager,
+            ModuleIdentifier{instrumented_function.file_path(),
```

Comment:
Nit: Can't you even omit the type here?

Reply:
Unfortunately `instrumented_function` is not a `FunctionInfo`, but `orbit_grpc_protos::InstrumentedFunction` and does not have a `module_id()` method. This could also be changed in the future.